tutorial/server.md

```diff
@@ -947,6 +947,186 @@ $ curl http://localhost:8081/foo
 not found
 ```
 
+## Nested APIs
+
+Let's take a break from introducing new combinators and functions and talk about how you can modularly define your *APIs*, avoiding as much as possible any kind of repetition. Consider this simple example:
```

Comment:
Just "Let's see how you can define your APIs in a modular way, while avoiding repetition. Consider..."?